ci

```diff
@@ -80,7 +80,7 @@ case "${1:?}"-"${2:?}" in
 
         cd deployment/ansible
         ansible-playbook -i inventory/openstack.yml --limit="epoch:&${DeployEnv:?}" \
-            --extra-vars "local_package=${FinalPackagePath:?} env=${DeployEnv:?} tester_keys_password='${tester_keys_password:?}'" \
+            --extra-vars "local_package=${FinalPackagePath:?} env=${DeployEnv:?} tester_keys_password='${tester_keys_password:?}' datadog_api_key=${datadog_api_key:?} datadog_app_key=${datadog_app_key:?}" \
```

Comment:
I assume no spaces hence no quotes.

Reply:
yes, AFAIK there is not spaces in datadog keys, however this will be revamped in https://www.pivotaltracker.com/story/show/154166551